s2n.mk

```diff
@@ -79,6 +79,11 @@ ifdef S2N_TEST_IN_FIPS_MODE
     DEFAULT_CFLAGS += -DS2N_TEST_IN_FIPS_MODE
 endif
 
+# Don't compile PQ related source code
+ifdef S2N_NO_PQ
+	DEFAULT_CFLAGS += -DS2N_NO_PQ
```

Comment:
Q: What is the intention behind `S2N_NO_PQ_ASM` and `S2N_NO_PQ` env variable? Do we want to enforce that if `S2N_NO_PQ_ASM` is set, we shouldn't be compiling PQ code.

Reply:
The `S2N_NO_PQ_ASM` is related to assembly optimization. We can still allow PQ code without optimization. If `S2N_NO_PQ` is set, then the optimization flag won't matter.